menu/server_configuration.sh

```diff
@@ -48,6 +48,7 @@ choice=$(whiptail --title "$TITLE" --checklist \
 $CHECKLIST_GUIDE\n\n$RUN_LATER_GUIDE" "$WT_HEIGHT" "$WT_WIDTH" 4 \
 "Static IP" "(Set static IP in Ubuntu with netplan.io)" OFF \
 "Security" "(Add extra security based on this http://goo.gl/gEJHi7)" OFF \
+"deSEC" "(Automatically get a domain from desec.io, configure ddclient and activate TLS)" ON \
```

Comment:
```suggestion
"deSEC" "(Automatically get a domain from desec.io, configure ddclient and activate TLS)" "$STARTUP_SWITCH" \
```
I think this makes more sense